multinet/db.py

```diff
@@ -138,6 +141,10 @@ def rename_workspace(old_name: str, new_name: str) -> None:
     coll = workspace_mapping_collection()
     coll.update(doc)
 
+    # Invalidate the cache for things changed by this function
+    get_workspace_db.cache_clear()
+    workspace_mapping.cache_clear()
```

Comment:
Does invalidating just the one workspace mapping cache cause the server to update the mapping_collection and its cache? It looks to me like we should invalidate the whole mapping_collection here instead, since it will now be out of sync with the server

Reply:
`workspace_mapping_collection` just returns the Collection itself, of type `StandardCollection`. Changes to any workspace mappings won't actually affect this, as you still have to call `find` on this collection to retrieve any data.

Reply:
Ah, thanks for the clarification.